tests/data/test_image.py

```diff
@@ -247,3 +247,17 @@ def numpy_reader(path):
         np.save(test_path, tensor)
         image = tio.ScalarImage(test_path, reader=numpy_reader)
         image.load()
+
```

Comment:
Add a test to check the required image attributes to be none

```suggestion
    def test_image_unload(self):
        path = self.get_image_path("unload")
        image = tio.ScalarImage(path)
        image.load()
        assert image._loaded
        image.unload()
        assert not image._loaded
        assert image[tio.constants.DATA] is None
        assert image[tio.constants.AFFINE] is None
```

Reply:
it shouldn't as the condition is
```python
if item in (DATA, AFFINE):
            if item not in self:
                self.load()
```
and `DATA` and `AFFINE` as keys are present in image, just set to None